index.js

```diff
@@ -1,22 +1,40 @@
 const takeUntil = sSrc => src => (start, sink) => {
-  let talkback, sTalkback;
+  if (start !== 0) return;
+  let sourceTalkback, sTalkback;
+  let inited = false;
```

Comment:
I'm hating this `initied`/`done` dance but we have to postpone `sink(2)` until it's actually greeted to cover sync pullable sources used as notifier